lib/CustomFields/pages/EditCustomFieldsRecord/EditCustomFieldsRecord.js

```diff
@@ -79,6 +81,17 @@ const EditCustomFieldsRecord = ({
   const formatedCustomFields = chunk(visibleCustomFields, columnCount);
   const columnWidth = rowShapes / columnCount;
 
+  const collectDataOptionsForSelect = (customField) => {
+    if (customField.type !== fieldTypes.SINGLE_SELECT_DROPDOWN) {
```

Comment:
I'd better move this check to line 107:
```js
dataOptions={customField.type !== fieldTypes.SINGLE_SELECT_DROPDOWN
  ? collectDataOptionsForSelect(customField)
  : null
}
```

Reply:
I agree. We also don't need to introduce `dataOptions` for other types. If it would be better to split up `renderCustomField` into two different methods. One for `SINGLE_SELECT_DROPDOWN` and another one for the reset of them.